.travis.yml

```diff
@@ -29,6 +29,7 @@ install:
   - composer install --no-interaction --no-progress --no-suggest --prefer-dist $COMPOSER_FLAGS
 
 script:
+  - ["$(ls _specs)" ] || echo "Empty"
```

Comment:
Tests that no files is in `_specs`. Indeed this directory should be used only for development but leaving files there is a mistake. It should however be extracted in a dedicated PR